CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Head
+
+- Fixed: missing `getTestRuleConfigs` export.
```

Comment:
nit

_add missing_ instead of _missing_

Reply:
I think keeping it as-is is better since "add" is confused with the "Added:" prefix.